### PR TITLE
JAMES-3028 Drop Swift configuration in profile

### PR DESCRIPTION
**Pull request by Arsnael**

### CODE REVIEW

File: stress-job.groovy [hunk 1]
```diff
@@ -5,7 +5,7 @@ pipelineJob("Stress-job") {
         stringParam('BRANCH', '', 'James branch')
         stringParam('SIMULATION', '', 'Gatling simulation')
         stringParam('GITHUB_SIMULATIONS', '', 'Github repository containing gatling simulations')
-        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), swift or s3')
+        stringParam('PROFILE', 's3', 'Blobs store target: s3 (default) or reference (for over-time performance tracking, but disabled for now)')
```
Comment (chibenwa) [inline stress-job.groovy]: Why do we need this profile ever again?

Comment (Arsnael) [reply stress-job.groovy]: Hmmm my guess would be when we rebuild the corpus with s3, to be able to choose between `reference` (so with corpus) and `s3` (without corpus) ? Could probably think of a better renaming though :)

File: stress-job.groovy [hunk 2]
```diff
@@ -5,7 +5,7 @@ pipelineJob("Stress-job") {
         stringParam('BRANCH', '', 'James branch')
         stringParam('SIMULATION', '', 'Gatling simulation')
         stringParam('GITHUB_SIMULATIONS', '', 'Github repository containing gatling simulations')
-        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), swift or s3')
+        stringParam('PROFILE', 'reference', 'Blobs store target: reference (default, for over-time performance tracking), s3 or s3-local')
```
Comment (Arsnael) [inline stress-job.groovy:8]: * `refrence` will be back as soon as the provisioning is finished
* `s3` will be like the Mokas did before, to some distant s3 ovh object storage... I will need to figure out how to access this
* `s3-local` : I added this to run a virgin s3 object-storage locally

Comment (chibenwa) [reply stress-job.groovy:8]: s3 needs some environment variable setup apparently

Comment (Arsnael) [reply stress-job.groovy:8]: That's supposed to be done if I understood correctly by the block I put back in the groovy stress test file in https://github.com/linagora/james-project/pull/3811/files#diff-586cbbd770bf7627b91cbdf94e9239baR48 (and that's what we should test and fix)

**Review by chibenwa — APPROVED** [review]

**Pull request merged**